package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "pdf-powerpoint",
```

Comment:
I was on the fence about this, but I think you're right.  Since there aren't a huge number of people using this library it is probably good to rename to something more generic.